tasks/main.yml

```
@@ -21,6 +21,18 @@
     dest: /etc/cron.weekly/virus_scan
     mode: 0755
 
+- name: Ensure that log folder exist
+  ansible.builtin.file:
+    path: /var/log/clamav
+    state: directory
+    mode: '0755'
+
+- name: Ensure log file will be world-readable
+  ansible.builtin.file:
+    path: /var/log/clamav/lastscan.log
+    state: file
+    mode: '0644'
```

Remove unnecessary quotes:
```suggestion
    mode: 0644
```